src/user/using-the-capture-app.md

```diff
@@ -848,6 +848,13 @@ When you deselect the enrollment you see the following
 
 ![](resources/images/capture_app/enrollment-dash-08.png)
 
+
+###Quick actions
+
+The quick actions-widget contains shortcuts to the most common actions performed by the user. 
```

Comment:
Two points here:
- This is user documentation, so I don't think we should refer to `the user`. 
- I don't think the hypen in -widget is necessary.

I suggest:
_The quick actions widget offers shortcuts for frequently used actions for the current enrollment._

Reply:
I like Joe's suggestion!